packages/vite/src/utils/index.ts

```diff
@@ -11,6 +13,39 @@ export function isCSS (file: string) {
   return IS_CSS_RE.test(file)
 }
 
+export function isVue (id: string, opts: { type?: Array<'template' | 'script' | 'style'> } = {}) {
```

Comment:
Can we use directly the util from the core ?

Reply:
`core/utils/plugins`

Reply:
It's a different package and I'm reluctant to expose the export. But maybe we can inline it instead